piv/yubikey-piv-manager.md

```diff
@@ -8,4 +8,4 @@ The [Yubikey PIV Manager](https://developers.yubico.com/yubikey-piv-manager) is
 
 The command-line utility is more powerful (e.g. allows overriding pin and touch default slot policies) but it also more prone to user errors. It's very easy to overwrite an existing key, for instance.
 
-After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](device-initialization).
+After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](../device-initialization//README.md).
```

Comment:
Oops, double slash -- can you squash your commits after that?

Reply:
Oops!! Done. Thanks again for your patience.

Reply:
And tested!
I suggest a .gitignore for _book and node_modules